tests/integration/Db/Dialect/Mysql/ModifyColumnCest.php

```diff
@@ -36,21 +36,43 @@ public function dbAdapterPdoMysqlModifyColumn(IntegrationTester $I)
                     [
                         'type'    => Column::TYPE_TIMESTAMP,
                         'default' => "CURRENT_TIMESTAMP",
-                        'notNull' => false,
+                        'notNull' => true,
```

Comment:
Could you please do not change previous test but add a new one.

Reply:
No, because previous was wrong. See https://github.com/phalcon/cphalcon/pull/14888